homeassistant/components/netatmo/binary_sensor.py

```diff
@@ -185,41 +115,41 @@ def update(self):
+            if self._sensor_type == "Someone known":
+                self._state = self._data.camera_data.someone_known_seen(
+                    cid=self._camera_id, exclude=self._timeout
                 )
-            elif self._sensor_name == "Someone unknown":
-                self._state = self._data.camera_data.someoneUnknownSeen(
-                    self._home, self._camera_name, self._timeout
+            elif self._sensor_type == "Someone unknown":
+                self._state = self._data.camera_data.someone_unknown_seen(
+                    cid=self._camera_id, exclude=self._timeout
                 )
-            elif self._sensor_name == "Motion":
-                self._state = self._data.camera_data.motionDetected(
-                    self._home, self._camera_name, self._timeout
+            elif self._sensor_type == "Motion":
+                self._state = self._data.camera_data.motion_detected(
+                    cid=self._camera_id, exclude=self._timeout
                 )
-        elif self._cameratype == "NOC":
-            if self._sensor_name == "Outdoor motion":
-                self._state = self._data.camera_data.outdoormotionDetected(
-                    self._home, self._camera_name, self._timeout
+        elif self._camera_type == "NOC":
+            if self._sensor_type == "Outdoor motion":
+                self._state = self._data.camera_data.outdoor_motion_detected(
+                    cid=self._camera_id, offset=self._timeout
                 )
-            elif self._sensor_name == "Outdoor human":
-                self._state = self._data.camera_data.humanDetected(
-                    self._home, self._camera_name, self._timeout
+            elif self._sensor_type == "Outdoor human":
+                self._state = self._data.camera_data.human_detected(
+                    cid=self._camera_id, offset=self._timeout
                 )
-            elif self._sensor_name == "Outdoor animal":
-                self._state = self._data.camera_data.animalDetected(
-                    self._home, self._camera_name, self._timeout
+            elif self._sensor_type == "Outdoor animal":
+                self._state = self._data.camera_data.animal_detected(
+                    cid=self._camera_id, offset=self._timeout
                 )
-            elif self._sensor_name == "Outdoor vehicle":
-                self._state = self._data.camera_data.carDetected(
-                    self._home, self._camera_name, self._timeout
+            elif self._sensor_type == "Outdoor vehicle":
+                self._state = self._data.camera_data.car_detected(
+                    cid=self._camera_id, offset=self._timeout
                 )
-        if self._sensor_name == "Tag Vibration":
-            self._state = self._data.camera_data.moduleMotionDetected(
-                self._home, self._module_name, self._camera_name, self._timeout
+        if self._sensor_type == "Tag Vibration":
```

Comment:
These checks are now done even if camera type was NOC or NACamera

Reply:
It always has been that way. I only changed the variable and method names.